### DISCUSSION TITLE
Deleting files not possible

### DISCUSSION THREAD

**Discussion by chrysenteron** [opengisch/QField#3927] [Q&A]
Hi,
With the upcoming switch to a paid plan, I'm trying to shrink my project as much as possible, so not as much paid storage will be needed. However, I'm not able to delete old file versions (Server Error 500). Is deletion of files still meant to be disabled or is this a bug? I feel like it has to be possible to delete stuff before the end of Beta so, so people won't be forced to pay for storage that is not actually needed. 
If this is intentional, how do you plan to handle this for people who switch to a paid plan while deleting files is not possible? Will billing be adjusted somehow to compensate for this?

**Comment by suricactus** [comment]
Hey, @chrysenteron, thanks for reporting this. Thanks for reporting this error. Luckily, it is something that we already detected ourselves, fix is implemented and it is going to be deployed tonight.

Actually the official way to delete your files is using QFieldSync. Using the Web UI you can delete your file versions, but not the file with all it's versions.

**Reply by chrysenteron** [reply]
Thanks @suricactus for your quick response. I'm not actually trying to delete the entire  file, just the old versions, which should be identical to the current ones anyway, since they're tif files that haven't actually ever been changed. 
Right now, I'm also not able to log in at all. Is there ongoing maintenance at the moment?

**Reply by chrysenteron** [reply]
Nevermind, I got in now
